tests/test_rag.py

```diff
@@ -39,3 +43,12 @@ def test_predict_server_manage_state(self):
     messages = [{"role": "human", "content": "What is 1 + 1?"}]
     new_messages = self.rag.chat(messages)
     assert len(new_messages) == 1
+
+  def test_upload_docs(self, caplog):
```

Comment:
Could we add a test case covering PDF/url as well,  using https://samples.clarifai.com/test_doc.pdf ?